### PR TITLE
Add German American Bank

### PR DESCRIPTION
**Pull request by DivyaB1**
Hi,
I'm adding 'German American Bank' - https://germanamerican.com/personal/

### CODE REVIEW

File: data/brands/amenity/bank.json [hunk 1]
```diff
@@ -21,6 +21,16 @@
     }
   },
   "items": [
+     {
+      "displayName": "German American Bank",
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "amenity": "bank",
+        "brand": "German American Bank",
```
Comment (bhousel) [inline data/brands/amenity/bank.json:29]:
```suggestion
        "brand": "German American",
```
Based on their branding, I would just call it "German American" in both the `brand` and `displayName` props.

Screenshot 2023-07-21 at 12 30 07 PM

File: data/brands/amenity/bank.json [hunk 2]
```diff
@@ -21,6 +21,16 @@
     }
   },
   "items": [
+     {
+      "displayName": "German American Bank",
```
Comment (bhousel) [inline data/brands/amenity/bank.json:25]:
```suggestion
      "displayName": "German American",
```

**Review by bhousel — CHANGES_REQUESTED** [review]
Thanks!  What do you think of using the simpler name?  Their branding doesn't seem to include the word "Bank"

**Comment by DivyaB1** [comment]
Hi,
Thanks, they referred themselves as 'German American Bank' in their 'About Us' page so I added it. I've corrected it now!
https://germanamerican.com/about/history/#:~:text=The%20history%20of-,German%20American%20Bank,-tells%20quite%20an

**Comment by bhousel** [comment]
Thanks @DivyaB1 👍

**Review by bhousel — APPROVED** [review]

**Pull request merged**